docs/modules/utils/combine_docs_examples/vectorstores.ipynb

```diff
@@ -719,13 +719,244 @@
     "docs[0]"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## OpenSearch\n",
+    "To run, you should have the opensearch instance up and running: [here](https://opensearch.org/docs/latest/install-and-configure/install-opensearch/index/)\n",
+    "`similarity_search` by default performs the Approximate k-NN Search which uses one of the several algorithms like lucene, nmslib, faiss recommended for\n",
```

Comment:
What is similarity_search?

Reply:
`similarity_search` is the public method which is common in all the vector stores and used by end users to provide their search query text and perform the search operation on their index.